### ISSUE TITLE
cmake/Templates/betr.cmake.in

### ISSUE THREAD

**Issue by bishtgautam** [BeTR-biogeochemistry-modeling/sbetr#3]
[cmake/Templates/betr.cmake.in](https://github.com/BeTR-biogeochemistry-modeling/sbetr/blob/master/cmake/Templates/betr.cmake.in) has values for Alquimia. Is it even required?

**Comment by bandre-ucar** [comment]
No, it was just a copy and paste job that I haven't gotten around to updating for betr/cesm integration.
